### ISSUE TITLE
GDScript implementation with new 3.1.1 audio features

### ISSUE THREAD

**Issue by regakakobigman** [Bauxitedev/spectral-visualizer#1]
New audio features were announced for 3.2: https://godotengine.org/article/godot-32-will-get-new-audio-features

However, 3.1.1 RC1 was released and it seems like these features were backported to 3.1.1: https://godotengine.org/article/release-candidate-godot-3-1-1-rc-1 Although 3.1.1 is only a release candidate for now, the audio features seem to be fully functional.

It would take some work to completely re-write the project in GDScript, but I think it could be a great example of how to use the new spectrum analyzer.

**Comment by Bauxitedev** [comment]
I agree, it would be nice to be able to do this without having to use C# and going through the hassle of installing nuget packages, etc. If I have the time I'll try to see if I can convert it to GDScript.
